### ISSUE TITLE
Card does not connect to the Eyefiserver2

### ISSUE THREAD

**Issue by blueslow** [dgrant/eyefiserver2#29]
Hi,

Apart from retreving the mac ID and the uppload key from the Windows xml settings file and close down the windows server is there anything more that has to be done for the card inorder to have it work with eyefiserver2?  E.g. where does one configure the card to comunicate with the machine that runs the eyefiserver2?

The only thing that I see in the log file is that the eyefiserver2 is started and listening on the port 59268. No connections from the card or upploading of pictures.
B.t.w I'm running the eyfiserver2 on linux mint 17.3, running python 2.7.6.

**Comment by arrproductions** [comment]
did you find a solution for this?   seem to be having the same issue... server seams to be running but no cards are connecting!

**Comment by blueslow** [comment]
Yes, sort of.  However i'm not sure what action that was the succesfull one. I did the following:
1 I changed the storage folder  with the eyefi setup in windows and  USB to sdcard adapter.
2 I terminated all eyefi programs in the windows machine
3 Restarted the eyefiserver on the Linux machine.
It is now working. I also double checked the configuration and that the directories had right permission for the eyefiserver and the user which eyefiserver was executing. Another thing to bear in mind is that stopping the eyefiserver with the script isn't allways working (use ps uax |grep eyefyserver, kill -i $pidofeyefyserver) and remove the pid file before restarting the eyefiserver. I have not been able to stop the eyefiserver correctly after it has received photo or avi.even though i patched according to the suggestions and git repository. Hope this helps.
